Modules/mmapmodule.c

```diff
@@ -922,6 +988,9 @@ static struct PyMethodDef mmap_object_methods[] = {
     {"write_byte",      (PyCFunction) mmap_write_byte_method,   METH_VARARGS},
     {"__enter__",       (PyCFunction) mmap__enter__method,      METH_NOARGS},
     {"__exit__",        (PyCFunction) mmap__exit__method,       METH_VARARGS},
+#ifdef MMAP_MMAP_MPROTECT_METHODDEF
```

Comment:
It will be always defined.

See the generated code:
```

#ifndef MMAP_MMAP_MPROTECT_METHODDEF
    #define MMAP_MMAP_MPROTECT_METHODDEF
#endif /* !defined(MMAP_MMAP_MPROTECT_METHODDEF) */
/*[clinic end generated code: output=7fad9c2a8d3fcb45 input=a9049054013a1b77]*/
```